web/src/components/CannabisLocationAlert.tsx

```diff
@@ -9,9 +8,7 @@ export const CannabisLocationAlert = (): ReactElement => {
   const { business } = useUserData();
 
   return business?.profileData.industryId === "cannabis" ? (
-    <Alert variant="warning">
-      <Content>{Config.profileDefaults.default.cannabisLocationAlert}</Content>
```

Comment:
Removed `<Content>` to eliminate a circular dependency issue. This should never be anything but a string so it's highly unlikely this would _need_ the Content component.

Reply:
Good call out